### PR TITLE
feat: divide production from testing for the forum component

### PR DESCRIPTION
**Pull request by Sembauke**

### CODE REVIEW

**Comment by Sembauke** [comment]
not sure why I have these extra commits, I only made one.

File: lib/ui/views/forum/forum-post/forum_post_viewmodel.dart [hunk 1]
```diff
@@ -156,7 +156,7 @@ class PostViewModel extends BaseViewModel {
   ) {
     List urlPart = url!.split('{size}');
     String avatarUrl = '';
-    String baseUrl = 'https://forum.freecodecamp.org';
+    String baseUrl = 'https://mobilefccinstance.com';
```
Comment (Nirajn2311) [inline lib/ui/views/forum/forum-post/forum_post_viewmodel.dart:159]: Shouldn't this url come from the `getCurrentUrl` function you defined?

**Comment by Nirajn2311** [comment]
> not sure why I have these extra commits, I only made one.

Squash those commits into one commit

File: lib/ui/views/forum/forum-post/forum_post_viewmodel.dart (hunk 1 above)
Comment (Sembauke) [reply lib/ui/views/forum/forum-post/forum_post_viewmodel.dart:159]: Yes this line can be removed

**Review by Nirajn2311 — APPROVED** [review]

**Pull request merged**